packages/go/analysis/ad/queries.go

```diff
@@ -130,6 +148,32 @@ func FetchActiveDirectoryTierZeroRoots(ctx context.Context, db graph.Database, d
 			attackPathRoots.AddSet(enforcedGPOs)
 		}
 
+		if (autoTagT0ParentObjectsFlag) {
```

Comment:
Would a path with an attackPathRoot with ACL inheritance enabled and alternating containers/ous in the chain be valid? Something like: 
`(ou1:OU)-[:Contains]->(c1:Container)-[:Contains]->(ou2:OU)-[:Contains]->(c2:Container)-[:Contains]->(b:Base {system_tags: "admin_tier_0", isaclprotected: false})`

Reply:
Great question - Yup, path would be valid. By disabling ACL inheritance you prevent ACL-based attacks, but it does not stop an attacker from linking a malicious GPO to the parent OU which enables a takeover of the child principals. GPOs can only be linked to OUs however.